src/main/java/persistence/sql/mapper/ColumnValue.java

```diff
@@ -0,0 +1,27 @@
+package persistence.sql.mapper;
+
+import utils.ReflectionUtil;
+
+import java.lang.reflect.Method;
+
+public class ColumnValue {
+    private final Object value;
+
+    public ColumnValue(final Object object, final String fieldName) {
+        this.value = generateValue(object, fieldName);
+    }
+
+    private Object generateValue(final Object object, final String fieldName) {
+        String getter = ReflectionUtil.generateGetterMethodName(fieldName);
+        try {
+            Method method = object.getClass().getMethod(getter);
+            return method.invoke(object);
+        } catch (Exception e) {
+            throw new RuntimeException(fieldName + "에 " + getter + "가 존재하지 않습니다.");
+        }
+    }
```

Comment:
getter를 리플렉션에 사용하는 방식도 괜찮지만 아래와 같이 사용해보는건 어떨까요? 😄 
만약 getter 메서드가 존재하지 않더라도 값을 안전하게 꺼낼 수 있을 것 같아요!
```suggestion
    private Object generateValue(final Object object, final String fieldName) {
        Class<?> clazz = object.getClass();
        try {
            Field field = clazz.getDeclaredField(fieldName);
            field.setAccessible(true);
            return field.get(object);
        } catch (Exception e) {
            throw new RuntimeException(e);
        }
    }
```